common/contract.ts

```diff
@@ -12,6 +12,7 @@ export type AnyContractType =
 export type Contract<T extends AnyContractType = AnyContractType> = {
   id: string
   slug: string // auto-generated; must be unique
+  url: string
```

Comment:
The purpose of this type is to represent the data in our database, so it would be incorrect to add this here.

Reply:
Thanks for the context! Agreed it makes sense to wait on this, given the discussion around URLs